File: Arrays and String/26. Remove Duplicates from Sorted Array/remove_duplicates.cpp
class Solution
{
public:
    int removeDuplicates(vector<int> &nums)
    {
        int idx = 0;
        int val = nums[0];

        if (nums.size() == 1)
        {
            return 1;
        }

        for (int i = 1; i < nums.size(); i++)
        {
            if (nums[i] == val)
            {
                continue;
            }
            else
            {
                idx++;
                nums[idx] = nums[i];
                val = nums[i];
            }
        }
        return idx + 1;
    }
};

File: 3. Arrays and String/26. Remove Duplicates from Sorted Array/remove_duplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        idx = 0
        val = nums[0]

        if len(nums) == 1:
            return 1

        for i in range(1, len(nums)):
            if nums[i] == val:
                continue
            else:
                idx += 1
                nums[idx] = nums[i]
                val = nums[i]
        return idx + 1